### PR TITLE
feat: add flag `--skip-preview` on the push command

### PR DESCRIPTION
**Pull request by y-lakhdar**
https://coveord.atlassian.net/browse/CDX-403

## Proposed changes

Add a flag to skip the confirmation prompt. Can be useful when the command is running in CI

## Testing

- [ ] Unit Tests:
    
- [x] Functionnal Tests:
    
- [x] Manual Tests:
    

-----
CDX-403

### CODE REVIEW

**Review by louis-bompart — APPROVED** [review]
👍

File: packages/cli/src/commands/org/config/push.ts [hunk 1]
```diff
@@ -18,15 +19,25 @@ export default class Push extends SnapshotBase {
 
   public static flags = {
     ...SnapshotBase.flags,
+    skipPreview: flags.boolean({
+      char: 'f',
```
Comment (olamothe) [inline packages/cli/src/commands/org/config/push.ts:23]: Why the `f` for `skipPreview` ? 🤔

**Review by olamothe — APPROVED** [review]

File: packages/cli/src/commands/org/config/push.ts (hunk 1 above)
Comment (y-lakhdar) [reply packages/cli/src/commands/org/config/push.ts:23]: My bad. I initially named the flag `--force` and forgot to rename the char.

Comment (y-lakhdar) [reply packages/cli/src/commands/org/config/push.ts:23]: Does `-s` makes sens?

**Pull request merged**